### ISSUE TITLE
Re-enable flaky servo tests

### ISSUE THREAD

**Issue by sjahr** [ros-planning/moveit2#1531]
### Description
The servo tests are flaky :/ for example
[test_servo_singularity](https://github.com/ros-planning/moveit2/blob/7d758de1b2f2904b8c85520129fa8d48aad93713/moveit_ros/moveit_servo/CMakeLists.txt#L212-L219) and need to be debugged.

```
$ ( source /home/runner/work/moveit2/moveit2/.work/upstream_ws/install/setup.bash && cd /home/runner/work/moveit2/moveit2/.work/target_ws && colcon test-result --verbose; )
build/moveit_servo/Testing/20220823-1522/Test.xml: 10 tests, 0 errors, 1 failure, 0 skipped
- test_launch_test_servo_singularity.test.py
  <<< failure message
    -- run_test.py: invoking following command in '/home/runner/work/moveit2/moveit2/.work/target_ws/build/moveit_servo':
     - ros2 test /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_ros/moveit_servo/test/launch/test_servo_singularity.test.py test_binary_dir:=/home/runner/work/moveit2/moveit2/.work/target_ws/build/moveit_servo --junit-xml=/home/runner/work/moveit2/moveit2/.work/target_ws/build/moveit_servo/test_results/moveit_servo/test_launch_test_servo_singularity.test.py.xunit.xml --package-name=moveit_servo
    Running with ROS_DOMAIN_ID 1
    ROS_DOMAIN_ID 1
    [INFO] [launch]: All log files can be found below /root/.ros/log/2022-08-23-15-23-46-774692-fec281f51165-25450
    [INFO] [launch]: Default logging verbosity is set to INFO
    test_gtest_run_complete (moveit_servo.TestGTestWaitForCompletion) ... Warning: Automatically adding attributes like self.proc_info to the test class will be deprecated in a future release.  Instead, add proc_info to the test method argument list to access the test object you need
    [INFO] [ros2_control_node-1]: process started with pid [25464]
    [INFO] [spawner-2]: process started with pid [25466]
    [INFO] [spawner-3]: process started with pid [25468]
    [INFO] [servo_node_main-4]: process started with pid [25478]
    [INFO] [component_container_mt-5]: process started with pid [25481]
    [ros2_control_node-1] [INFO] [1661268227.108956772] [resource_manager]: Loading hardware 'PandaFakeSystem' 
    [ros2_control_node-1] [INFO] [1661268227.113878570] [resource_manager]: Initialize hardware 'PandaFakeSystem' 
    [ros2_control_node-1] [WARN] [1661268227.114187770] [mock_generic_system]: Parsing of optional initial interface values failed or uses a deprecated format. Add initial values for every state interface in the ros2_control.xacro. For example: 
    [ros2_control_node-1] <state_interface name="velocity"> 
    [ros2_control_node-1]   <param name="initial_value">0.0</param> 
    [ros2_control_node-1] </state_interface>
    [ros2_control_node-1] [INFO] [1661268227.116079169] [resource_manager]: Successful initialization of hardware 'PandaFakeSystem'
    [ros2_control_node-1] [INFO] [1661268227.117219069] [resource_manager]: Loading hardware 'PandaHandFakeSystem' 
    [ros2_control_node-1] [INFO] [1661268227.118885668] [resource_manager]: Initialize hardware 'PandaHandFakeSystem' 
    [ros2_control_node-1] [INFO] [1661268227.119564068] [resource_manager]: Successful initialization of hardware 'PandaHandFakeSystem'
    [ros2_control_node-1] [INFO] [1661268227.120353667] [resource_manager]: 'configure' hardware 'PandaFakeSystem' 
    [ros2_control_node-1] [INFO] [1661268227.120932367] [resource_manager]: Successful 'configure' of hardware 'PandaFakeSystem'
    [ros2_control_node-1] [INFO] [1661268227.121456067] [resource_manager]: 'activate' hardware 'PandaFakeSystem' 
    [ros2_control_node-1] [INFO] [1661268227.123076766] [resource_manager]: Successful 'activate' of hardware 'PandaFakeSystem'
    [ros2_control_node-1] [INFO] [1661268227.123747566] [resource_manager]: 'configure' hardware 'PandaHandFakeSystem' 
    [ros2_control_node-1] [INFO] [1661268227.124395265] [resource_manager]: Successful 'configure' of hardware 'PandaHandFakeSystem'
    [ros2_control_node-1] [INFO] [1661268227.125165265] [resource_manager]: 'activate' hardware 'PandaHandFakeSystem' 
    [ros2_control_node-1] [INFO] [1661268227.125713165] [resource_manager]: Successful 'activate' of hardware 'PandaHandFakeSystem'
    [ros2_control_node-1] [INFO] [1661268227.153255253] [controller_manager]: update rate is 100 Hz
    [ros2_control_node-1] [INFO] [1661268227.157625251] [controller_manager]: RT kernel is recommended for better performance
    [servo_node_main-4] [WARN] [1661268227.251882311] [moveit_servo.servo_node]: Intra-process communication is disabled, consider enabling it by adding: 
    [servo_node_main-4] extra_arguments=[{'use_intra_process_comms' : True}]
    [servo_node_main-4] to the Servo composable node in the launch file
    [servo_node_main-4] [INFO] [1661268227.261027407] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 0.002761 seconds
    [servo_node_main-4] [INFO] [1661268227.261069507] [moveit_robot_model.robot_model]: Loading robot model 'panda'...
    [spawner-2] [INFO] [1661268227.267630204] [spawner_joint_state_broadcaster]: Waiting for '/controller_manager' node to exist
    [servo_node_main-4] Link panda_link1 had 1 children
    [servo_node_main-4] Link panda_link2 had 1 children
    [servo_node_main-4] Link panda_link3 had 1 children
    [servo_node_main-4] Link panda_link4 had 1 children
    [servo_node_main-4] Link panda_link5 had 1 children
    [servo_node_main-4] Link panda_link6 had 1 children
    [servo_node_main-4] Link panda_link7 had 1 children
    [servo_node_main-4] Link panda_link8 had 1 children
    [servo_node_main-4] Link panda_hand had 2 children
    [servo_node_main-4] Link panda_leftfinger had 0 children
    [servo_node_main-4] Link panda_rightfinger had 0 children
    [component_container_mt-5] [INFO] [1661268227.280309199] [test_servo_integration_container]: Load Library: /opt/ros/humble/lib/librobot_state_publisher_node.so
    [component_container_mt-5] [INFO] [1661268227.283291797] [test_servo_integration_container]: Found class: rclcpp_components::NodeFactoryTemplate<robot_state_publisher::RobotStatePublisher>
    [component_container_mt-5] [INFO] [1661268227.283333497] [test_servo_integration_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<robot_state_publisher::RobotStatePublisher>
    [component_container_mt-5] Link panda_link1 had 1 children
    [component_container_mt-5] Link panda_link2 had 1 children
    [component_container_mt-5] Link panda_link3 had 1 children
    [component_container_mt-5] Link panda_link4 had 1 children
    [component_container_mt-5] Link panda_link5 had 1 children
    [component_container_mt-5] Link panda_link6 had 1 children
    [component_container_mt-5] Link panda_link7 had 1 children
    [component_container_mt-5] Link panda_link8 had 1 children
    [component_container_mt-5] Link panda_hand had 2 children
    [component_container_mt-5] Link panda_leftfinger had 0 children
    [component_container_mt-5] Link panda_rightfinger had 0 children
    [component_container_mt-5] [INFO] [1661268227.313109584] [robot_state_publisher]: got segment panda_hand
    [component_container_mt-5] [INFO] [1661268227.315515483] [robot_state_publisher]: got segment panda_leftfinger
    [component_container_mt-5] [INFO] [1661268227.315541583] [robot_state_publisher]: got segment panda_link0
    [component_container_mt-5] [INFO] [1661268227.315549083] [robot_state_publisher]: got segment panda_link1
    [component_container_mt-5] [INFO] [1661268227.315555083] [robot_state_publisher]: got segment panda_link2
    [component_container_mt-5] [INFO] [1661268227.315560883] [robot_state_publisher]: got segment panda_link3
    [component_container_mt-5] [INFO] [1661268227.315566583] [robot_state_publisher]: got segment panda_link4
    [component_container_mt-5] [INFO] [1661268227.315572183] [robot_state_publisher]: got segment panda_link5
    [component_container_mt-5] [INFO] [1661268227.315578183] [robot_state_publisher]: got segment panda_link6
    [component_container_mt-5] [INFO] [1661268227.315583883] [robot_state_publisher]: got segment panda_link7
    [component_container_mt-5] [INFO] [1661268227.315589383] [robot_state_publisher]: got segment panda_link8
    [component_container_mt-5] [INFO] [1661268227.315595483] [robot_state_publisher]: got segment panda_rightfinger
    [INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/robot_state_publisher' in container '/test_servo_integration_container'
    [component_container_mt-5] [INFO] [1661268227.326328079] [test_servo_integration_container]: Load Library: /opt/ros/humble/lib/libstatic_transform_broadcaster_node.so
    [component_container_mt-5] [INFO] [1661268227.326638479] [test_servo_integration_container]: Found class: rclcpp_components::NodeFactoryTemplate<tf2_ros::StaticTransformBroadcasterNode>
    [component_container_mt-5] [INFO] [1661268227.326657779] [test_servo_integration_container]: Instantiate class: rclcpp_components::NodeFactoryTemplate<tf2_ros::StaticTransformBroadcasterNode>
    [INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/static_tf2_broadcaster' in container '/test_servo_integration_container'
    [servo_node_main-4] [INFO] [1661268227.358380365] [moveit_ros.current_state_monitor]: Listening to joint states on topic '/joint_states'
    [servo_node_main-4] [INFO] [1661268227.364394762] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/attached_collision_object' for attached collision objects
    [servo_node_main-4] [INFO] [1661268227.364861162] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting planning scene monitor
    [servo_node_main-4] [INFO] [1661268227.366887461] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/planning_scene'
    [servo_node_main-4] [INFO] [1661268227.368058961] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Publishing maintained planning scene on '/servo_node/publish_planning_scene'
    [servo_node_main-4] [WARN] [1661268227.368488961] [moveit_ros.current_state_monitor]: Unable to update multi-DOF joint 'virtual_joint':Failure to lookup transform between 'world'and 'panda_link0' with TF exception: "world" passed to lookupTransform argument target_frame does not exist. 
    [ros2_control_node-1] [INFO] [1661268227.386292553] [controller_manager]: Loading controller 'panda_arm_controller'
    [ros2_control_node-1] [INFO] [1661268227.398997048] [joint_trajectory_controller.interpolation_methods]: No interpolation method parameter was given. Using the default, VARIABLE_DEGREE_SPLINE.
    [spawner-3] [INFO] [1661268227.422103938] [spawner_panda_arm_controller]: [NON-XML-CHAR-0x1B][94mLoaded panda_arm_controller[NON-XML-CHAR-0x1B][0m
    [ros2_control_node-1] [INFO] [1661268227.423672137] [controller_manager]: Configuring controller 'panda_arm_controller'
    [ros2_control_node-1] [INFO] [1661268227.423884837] [panda_arm_controller]: Command interfaces are [position] and state interfaces are [position velocity].
    [ros2_control_node-1] [INFO] [1661268227.425885136] [panda_arm_controller]: Controller state will be published at 50.00 Hz.
    [ros2_control_node-1] [INFO] [1661268227.429206635] [panda_arm_controller]: Action status changes will be monitored at 20.00 Hz.
    [spawner-3] [INFO] [1661268227.448732426] [spawner_panda_arm_controller]: [NON-XML-CHAR-0x1B][92mConfigured and activated [NON-XML-CHAR-0x1B][96mpanda_arm_controller[NON-XML-CHAR-0x1B][0m
    [ros2_control_node-1] [INFO] [1661268227.474952415] [controller_manager]: Loading controller 'joint_state_broadcaster'
    [spawner-2] [INFO] [1661268227.488920609] [spawner_joint_state_broadcaster]: [NON-XML-CHAR-0x1B][94mLoaded joint_state_broadcaster[NON-XML-CHAR-0x1B][0m
    [ros2_control_node-1] [INFO] [1661268227.490369608] [controller_manager]: Configuring controller 'joint_state_broadcaster'
    [ros2_control_node-1] [INFO] [1661268227.490465008] [joint_state_broadcaster]: 'joints' or 'interfaces' parameter is empty. All available state interfaces will be published
    [spawner-2] [INFO] [1661268227.508949300] [spawner_joint_state_broadcaster]: [NON-XML-CHAR-0x1B][92mConfigured and activated [NON-XML-CHAR-0x1B][96mjoint_state_broadcaster[NON-XML-CHAR-0x1B][0m
    [INFO] [spawner-3]: process has finished cleanly [pid 25468]
    [INFO] [spawner-2]: process has finished cleanly [pid 25466]
    [servo_node_main-4] [WARN] [1661268228.398309118] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [INFO] [test_servo_singularity-6]: process started with pid [25534]
    [test_servo_singularity-6] [==========] Running 1 test from 1 test suite.
    [test_servo_singularity-6] [----------] Global test environment set-up.
    [test_servo_singularity-6] [----------] 1 test from ServoFixture
    [test_servo_singularity-6] [ RUN      ] ServoFixture.ReachSingular
    [servo_node_main-4] [WARN] [1661268229.428284376] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268230.448949505] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268231.488404712] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268232.518192385] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268233.538239923] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268234.538339131] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268235.558013607] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268236.558043105] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268237.568158491] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268238.578955354] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [INFO] [1661268239.505669655] [moveit_servo.servo_calcs]: RT kernel is recommended for better performance
    [test_servo_singularity-6] [INFO] [1661268239.506363556] [moveit_servo.servo_launch_test_common.hpp]: Wait for start servo: 10.018
    [test_servo_singularity-6] [INFO] [1661268239.556530615] [moveit_servo.servo_launch_test_common.hpp]: Wait for status num increasing: 0.0500832
[servo_node_main-4] [WARN] [1661268239.598953865] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268239.609412078] [moveit_servo.servo_calcs]: Moving away from a singularity, decelerating
    [test_servo_singularity-6] [INFO] [1661268239.656659234] [moveit_servo.servo_launch_test_common.hpp]: Wait for DECELERATE_FOR_LEAVING_SINGULARITY: 0.100072
    [servo_node_main-4] [WARN] [1661268240.638196500] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268241.658096711] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268242.635126871] [moveit_servo.servo_calcs]: Moving away from a singularity, decelerating
    [servo_node_main-4] [WARN] [1661268242.658398298] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268243.678453310] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [test_servo_singularity-6] [INFO] [1661268244.006662799] [moveit_servo.servo_launch_test_common.hpp]: Wait for NO_WARNING: 4.34995
    [test_servo_singularity-6] [INFO] [1661268244.256661496] [moveit_servo.servo_launch_test_common.hpp]: Wait for DECELERATE_FOR_APPROACHING_SINGULARITY: 0.15
    [servo_node_main-4] [WARN] [1661268244.698961922] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268245.661131364] [moveit_servo.servo_calcs]: Moving closer to a singularity, decelerating
    [servo_node_main-4] [WARN] [1661268245.718319232] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268246.748242618] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268247.767888017] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268248.687162325] [moveit_servo.servo_calcs]: Moving closer to a singularity, decelerating
    [servo_node_main-4] [WARN] [1661268248.768519837] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268249.788244336] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268250.788959756] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268251.713134159] [moveit_servo.servo_calcs]: Moving closer to a singularity, decelerating
    [servo_node_main-4] [WARN] [1661268251.808959155] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268252.828135355] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [servo_node_main-4] [WARN] [1661268253.828141976] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [test_servo_singularity-6] /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_ros/moveit_servo/test/test_servo_singularity.cpp:145: Failure
    [test_servo_singularity-6] Expected: (iterations) < (test_parameters_->timeout_iterations), actual: 201 vs 200
    [test_servo_singularity-6] [INFO] [1661268254.256718304] [moveit_servo.servo_launch_test_common.hpp]: Wait for HALT_FOR_SINGULARITY: 10
    [test_servo_singularity-6] [  FAILED  ] ServoFixture.ReachSingular (25146 ms)
    [test_servo_singularity-6] [----------] 1 test from ServoFixture (25146 ms total)
    [test_servo_singularity-6] 
    [test_servo_singularity-6] [----------] Global test environment tear-down
    [test_servo_singularity-6] [==========] 1 test from 1 test suite ran. (25146 ms total)
    [test_servo_singularity-6] [  PASSED  ] 0 tests.
    [test_servo_singularity-6] [  FAILED  ] 1 test, listed below:
    [test_servo_singularity-6] [  FAILED  ] ServoFixture.ReachSingular
    [test_servo_singularity-6] 
    [test_servo_singularity-6]  1 FAILED TEST
Error: ROR] [test_servo_singularity-6]: process has died [pid 25534, exit code 1, cmd '/home/runner/work/moveit2/moveit2/.work/target_ws/build/moveit_servo/test_servo_singularity --ros-args --params-file /tmp/launch_params_1tw7l359'].
    ok
    
    ----------------------------------------------------------------------
    Ran 1 test in 27.338s
    
    OK
    [INFO] [component_container_mt-5]: sending signal 'SIGINT' to process[component_container_mt-5]
    [INFO] [servo_node_main-4]: sending signal 'SIGINT' to process[servo_node_main-4]
    [INFO] [ros2_control_node-1]: sending signal 'SIGINT' to process[ros2_control_node-1]
    [servo_node_main-4] [WARN] [1661268254.828376507] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
    [component_container_mt-5] [INFO] [1661268255.358775213] [rclcpp]: signal_handler(signum=2)
    [ros2_control_node-1] [INFO] [1661268255.361010502] [rclcpp]: signal_handler(signum=2)
    [servo_node_main-4] [INFO] [1661268255.363464790] [rclcpp]: signal_handler(signum=2)
    [servo_node_main-4] Warning: class_loader.ClassLoader: SEVERE WARNING!!! Attempting to unload library while objects created by this loader exist in the heap! You should delete your objects before attempting to unload the library or destroying the ClassLoader. The library will NOT be unloaded.
    [servo_node_main-4]          at line 127 in ./src/class_loader.cpp
    [servo_node_main-4] [INFO] [1661268255.373151342] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Stopped publishing maintained planning scene.
    [servo_node_main-4] [INFO] [1661268255.376632125] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Stopping planning scene monitor
    [INFO] [ros2_control_node-1]: process has finished cleanly [pid 25464]
    [INFO] [servo_node_main-4]: process has finished cleanly [pid 25478]
Error: ROR] [component_container_mt-5]: process has died [pid 25481, exit code -11, cmd '/opt/ros/humble/lib/rclcpp_components/component_container_mt --ros-args -r __node:=test_servo_integration_container -r __ns:=/'].
    test_gtest_pass (moveit_servo.TestGTestProcessPostShutdown) ... FAIL
    
    ======================================================================
    FAIL: test_gtest_pass (moveit_servo.TestGTestProcessPostShutdown)
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_ros/moveit_servo/test/launch/test_servo_singularity.test.py", line 27, in test_gtest_pass
        launch_testing.asserts.assertExitCodes(proc_info, process=servo_gtest)
      File "/opt/ros/humble/lib/python3.10/site-packages/launch_testing/asserts/assert_exit_codes.py", line 62, in assertExitCodes
        assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(
    AssertionError: Proc test_servo_singularity-6 exited with code 1
    
    ----------------------------------------------------------------------
    Ran 1 test in 0.000s
    
    FAILED (failures=1)
    -- run_test.py: return code 1
    -- run_test.py: verify result file '/home/runner/work/moveit2/moveit2/.work/target_ws/build/moveit_servo/test_results/moveit_servo/test_launch_test_servo_singularity.test.py.xunit.xml'
  >>>
build/moveit_servo/test_results/moveit_servo/test_launch_test_servo_singularity.test.py.xunit.xml: 2 tests, 0 errors, 1 failure, 0 skipped
- moveit_servo.TestGTestProcessPostShutdown test_gtest_pass
  <<< failure message
    Traceback (most recent call last):
      File "/home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_ros/moveit_servo/test/launch/test_servo_singularity.test.py", line 27, in test_gtest_pass
        launch_testing.asserts.assertExitCodes(proc_info, process=servo_gtest)
      File "/opt/ros/humble/lib/python3.10/site-packages/launch_testing/asserts/assert_exit_codes.py", line 62, in assertExitCodes
        assert info.returncode in allowable_exit_codes, 'Proc {} exited with code {}'.format(
    AssertionError: Proc test_servo_singularity-6 exited with code 1
  >>>
```

**Comment by sjahr** [comment]
Maybe addressed by #1543

**Comment by sjahr** [comment]
Apparently, it fixed it :)
